content/02.introduction.md

```diff
@@ -104,9 +105,54 @@ It also includes cell-surface receptors that can recognize the molecular pattern
 These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
 After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
-One of the main immune responses that contribute to the onset of acute respiratory distress syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
-In addition to respiratory distress, this mechanism can lead to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
-Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.
+
+There are several ways in which the SARS-CoV virus can evade the human immune response.
+One way is through CD163+ and CD68+ macrophage cells, which are crucial for the establishment of SARS in the body [@doi:10.1038/mi.2015.127].
+These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
+According to a study on the viral dissemination of SARS-CoV in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells [@doi:10.1038/mi.2015.127].
+This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of detectable viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
+Another strategy that SARS-CoV utilizes to increase its virulence is the suppression of host gene expression by nonstructural protein 1 (Nsp1).
+Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
+These evasion mechanisms, in turn, can lead to systemic infection.
+
+SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
+SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
+MERS-CoV also utilizes these mechanisms, as well as histone modification.
+Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].
+
+Based on the aforementioned mechanisms for both SARS-CoV and MERS-CoV, SARS-CoV-2 most likely utilizes similar mechanisms for immune evasion.
```

Comment:
I would put a citation here

Reply:
I can't seem to edit this here, so linking it into #349